l10n/DataAccess/Models/MetaDataEntity.cs

```diff
@@ -0,0 +1,22 @@
+﻿using System;
+using System.ComponentModel.DataAnnotations;
+
+namespace DataAccess.Models
+{
+    public class MetaDataEntity
+    {
+        /// <summary>
+        /// The date and time for when the user was first created
+        /// </summary>
+        [Required]
+        public DateTime Created { get; set; }
+
+        /// <summary>
+        /// Automatically set on insert or update.
+        /// The property is also treated as a concurrency token. This ensures that an exception is thrown if anyone else has modified this row since the row was queried.
+        /// </summary>
+        [Timestamp]
```

Comment:
This is to use optimistic concurrency control. Related: https://docs.microsoft.com/en-us/ef/core/modeling/concurrency#timestamprow-version

Reply:
Nice way of handling it